charts/delivery/Chart.yaml

```
@@ -1,4 +1,13 @@
+apiVersion: v2
 name: delivery
 version: v0.1.0
 appVersion: v0.1.0
 description: Fabrikam Drone Delivery Service
+dependencies:
+  - name: delivery-dev
+    repository: "file://envs/delivery-dev"
+    version: ">= 0.0.1"
```

```suggestion
    version: "v0.1.0"
    condition: envs.dev
```

please let's propagate this change to the rest of the charts.

---

I think this change needs to be replicated on every `chart.yaml` if I am not mistaken.

---

You are right, just updated the rest of the charts and tested.